auth_proxy/models/oauth.py

```diff
@@ -21,8 +21,11 @@ class Client(Base):
     client_id = Column(String, primary_key=True)
     client_secret = Column(String, nullable=False)
 
-    _redirect_uris = Column(Text)
-    _default_scopes = Column(Text)
+    user_id = Column(Integer, ForeignKey('user.id'))
+    user = orm.relationship('User')
```

Comment:
I'm having trouble understanding the relationship between `client` and `user`. Normally a client would be registered with a server outside of any particular user context. And then multiple users could approve that client -- the user isn't (shouldn't be) a property of the client.